pkg/controller/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -139,6 +139,12 @@ func (r *ReconcilePublishingStrategy) Reconcile(request reconcile.Request) (reco
 		}
 	}
 
+	// ----------------------------TODO: remove these debug logs afterwards --------------------------------------
+	log.Info(fmt.Sprintf("initial ingresscontroller list: %v", ingressControllerList.Items))
+	log.Info(fmt.Sprintf("appingress on publishingstrategy CR: %v", instance.Spec.ApplicationIngress))
+	log.Info(fmt.Sprintf("ingress to reconcile: %v", ingressNotOnCluster))
```

Comment:
```suggestion
	log.Info(fmt.Sprintf("initial ingresscontroller list: %+v", ingressControllerList.Items))
	log.Info(fmt.Sprintf("appingress on publishingstrategy CR: %+v", instance.Spec.ApplicationIngress))
	log.Info(fmt.Sprintf("ingress to reconcile: %+v", ingressNotOnCluster))
```

`%+v` will probably be a bit better to print field names.